cmake/Modules/winconf-mingw.cmake

```diff
@@ -6,4 +6,5 @@ list(APPEND LibtorrentRasterbar_CUSTOM_DEFINITIONS
     -D_FILE_OFFSET_BITS=64
     -D__USE_W32_SOCKETS)
 
-link_libraries(advapi32 iphlpapi ole32 powrprof shell32 user32 wsock32 ws2_32)
+# libraries from winconf.pri
+link_libraries(advapi32 iphlpapi ole32 powrprof shell32 user32 wsock32 ws2_32)
```

Comment:
Make sure there is an empty line at end of the file, both files.

Reply:
done?